### DISCUSSION TITLE
Problem

### DISCUSSION THREAD

**Discussion by inquisitor-svg** [jassielcastro/ChessApp#9] [General]
When I try to create board, it failed..... My ide remind me I have to create class board

**Comment by jassielcastro** [comment]
Hi! Could you tell me what version of the library you are using and if you are sure that the library was downloaded?
Do you have access to the library packages?
I mean if when you synchronize your project the Build finished without warnings.
Something like this:

<img width="1064" alt="Screen Shot 2022-10-27 at 1 10 01" src="https://user-images.githubusercontent.com/34942375/198204979-e4213724-a513-4be6-a880-223d74ce4ef7.png">

**Reply by inquisitor-svg** [reply]
Thanks for reply, yes, the problem is here. 
![屏幕截图(263)](https://user-images.githubusercontent.com/82759374/198223036-efa69d3b-d942-4c55-9ad9-dcf379cdac48.png)

**Reply by inquisitor-svg** [reply]
I have no idea how to solve this, I think the current version is 1.0.1

**Comment by inquisitor-svg** [comment]
Thx, I finally debbuged successfully, thx for your help

Alan Jassiel Castro Melgoza ***@***.***> 于2022年10月27日周四
14:10写道：
